.codesandbox/sandbox/end-to-end-test.html

```diff
@@ -10,7 +10,7 @@
 	<input type="url" name="idp" id="idp" value="https://broker.pod.inrupt.com"/>
 	<button>Log in</button>
 
-	<script src="src/end-to-end-test.ts">
+	<script src="src/end-to-end-test.ts" type="module">
```

Comment:
This is apparently needed now to tell Parcel to correctly parse `import` statements.